__tests__/__snapshots__/snapshot.tsx.snap

```diff
@@ -0,0 +1,103 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Snapshot tests are a way to ensure that a component doesn't render unexpectedly without developers intentionally implementing a new change.

I'm not a huge fan of this type of test, but it's part of the jest default setup. If a snapshot test ever fails after you introduce new changes, just run `npm run test:updateSnapshot` to generate a new snapshot. Then the test will pass again.